CONTRIBUTING.md

```diff
@@ -127,27 +127,84 @@ mvn test
-`integration_apikey.txt` at the root of the codebase.
-
-Some of the integration tests require an
-[Identity Platform](https://cloud.google.com/identity-platform/) project with multi-tenancy
-[enabled](https://cloud.google.com/identity-platform/docs/multi-tenancy-quickstart#enabling_multi-tenancy).
-An existing Firebase project can be upgraded to an Identity Platform project without losing any
-functionality via the
-[Identity Platform Marketplace Page](https://console.cloud.google.com/customer-identity). Note that
-charges may be incurred for active users beyond the Identity Platform free tier.
+tests.
+
+Integration tests are executed against a real life Firebase project. If you do not already
+have one suitable for running the tests against, you can create a new project in the
+[Firebase Console](https://console.firebase.google.com) following the setup guide below.
+If you already have a Firebase project, you'll need to obtain credentials to communicate and
+authorize access to your Firebase project:
+
+1. Service account certificate: This allows access to your Firebase project through a service account
+which is required for all integration tests. This can be downloaded as a JSON file from the 
+**Settings > Service Accounts** tab of the Firebase console when you click the
+**Generate new private key** button. Copy the file into the repo so it's available at
+`integration_cert.json`.
+
+
+2. Web API key: This allows for Auth sign-in needed for some Authentication and Tenant Management
+integration tests. This is displayed in the **Settings > General** tab of the Firebase console
+after enabling Authentication as described in the steps below. Copy it and save to a new text
+file at `integration_apikey.txt`.
+
+
+Set up your Firebase project as follows:
+
+
+1. Enable Authentication:
+   1. Go to the Firebase Console, and select **Authentication** from the **Build** menu.
+   2. Click on **Get Started**.
+   3. Select **Sign-in method > Add new provider > Email/Password** then enable both the
+   **Email/Password** and **Email link (passwordless sign-in)** options.
+
+
+2. Enable Firestore:
+   1. Go to the Firebase Console, and select **Firestore Database** from the **Build** menu.
+   2. Click on the **Create database** button. You can choose to set up Firestore either in
+   the production mode or in the test mode.
+
+
+3. Enable Realtime Database:
+   1. Go to the Firebase Console, and select **Realtime Database** from the **Build** menu.
+   2. Click on the **Create Database** button. You can choose to set up the Realtime Database
+   either in the locked mode or in the test mode.
+   3. In the **Data** tab click on the kebab menu (3 dots) and select **Create Database**.
+   4. Enter your Project ID (Found in the **General** tab in **Account Settings**) as the
+   **Realtime Database reference**. Again, you can choose to set up the Realtime Database
+   either in the locked mode or in the test mode.
```

Comment:
I do what to highlight that this portion is asking to create 2 separate Realtime Databases. 
- The first being the default one which has a different naming convention than what is used in our integration tests.
- The second being named using the project id of the firebase project which is the naming convention in our integration tests.

The reason for this is users are not allowed to enter a custom name for their first rtdb and changing the integration test references will break current test projects. This will be confusing so I wondering if I should go into detail here as to why.

Reply:
Definitely -- I didn't quite gather that from reading the steps (it's possible I was confused because we talk about the a database "ref" value for the targeted part of a path in CF3, but that's not really relevant here).

Yup, I would definitely add some text to clarify that.